unstructured/ingest/cli/cmds/salesforce.py

```diff
@@ -0,0 +1,62 @@
+import logging
+
+import click
+
+from unstructured.ingest.cli.common import (
+    add_recursive_option,
+    add_shared_options,
+    log_options,
+    map_to_processor_config,
+    map_to_standard_config,
+    run_init_checks,
+)
+from unstructured.ingest.logger import ingest_log_streaming_init, logger
+from unstructured.ingest.runner import salesforce as salesforce_fn
+
+
+@click.command()
+@click.option(
+    "--salesforce-categories",
```

Comment:
Given the cli command is `salesforce`, I usually find having `salesforce` as part of the params redundant.

Reply:
@rbiseck3  - I hear you. Back before the refactor, this wasn't the case. But I would definitely prefer to simplify. Thank you.

Reply:
ah, yea, good catch here.